Parsers/ASimWebSession/Parsers/ASimWebSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,452 @@
+Parser:
+  Title: Web Session ASIM parser for SonicWall firewalls
+  Version: '0.1.0'
+  LastUpdated: Jan 26, 2024
+Product:
+  Name: SonicWall
+Normalization:
+  Schema: WebSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Web Session Schema
+  Link: https://aka.ms/ASimWebSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing SonicWall SonicOS ArcSight-formatted syslog ingested by Microsoft Sentinel to the ASIM WebSession normalized schema.
+ParserName: ASimWebSessionSonicWallFirewall
+EquivalentBuiltInParser: _ASim_WebSession_SonicWallFirewall
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser=(disabled:bool=false){
+      let Actions=datatable(fw_action:string, DvcAction:string, EventSeverity:string)
+        [ "reset client", "Reset Source", "Low"
+        , "reset server", "Reset Destination", "Low"
+        , "reset both", "Reset", "Low"
+        , "allow", "Allow", "Informational"
+        , "\"forward\"", "Allow", "Informational"
+        , "\"mgmt\"", "Other", "Informational"
+        , "\"NA\"", "Other", "Informational"
+        , "deny", "Deny", "Low"
+        , "\"drop\"", "Drop", "Low"
+        , "drop ICMP", "Drop ICMP", "Low"

```

Comment:
@jaimeesc can you confirm the fw_action values in case of Web session or URL logs are matching with what you configured above.

Reply:
I was just recycling that section from the Network Session parser since the expected fw_action values are the same. I'll remove the others that I haven't seen in my logs for web sessions.